docs/sample_config.yaml

```diff
@@ -93,6 +93,23 @@ pid_file: DATADIR/homeserver.pid
 #
 #public_baseurl: https://example.com/
 
+# Uncomment the following to tell other servers to redirect traffic to port 443.
```

Comment:
```suggestion
# Uncomment the following to tell other servers to communicate on port 443.
```

'redirect' reads a little bit wrong here to me; it also brings to mind HTTP redirects and I can see it giving people some imagery that 8448 will serve redirects to 443 or something.

Reply:
done, thanks.